### PR TITLE
Add test case for createPathFind and convert snake case function calls to camel case

### PR DESCRIPTION
**Pull request by clark800**

### CODE REVIEW

**Comment by clark800** [comment]
This is intended to resolve: https://ripplelabs.atlassian.net/browse/RLJS-273

File: src/core/pathfind.js [hunk 1]
```diff
@@ -54,16 +58,16 @@ PathFind.prototype.create = function () {
...
   this.remote.request_path_find_close().broadcast().request();
   this.emit('end');
   this.emit('close');
 };
 
-PathFind.prototype.notify_update = function (message) {
-  var src_account = message.source_account;
-  var dst_account = message.destination_account;
-  var dst_amount  = Amount.from_json(message.destination_amount);
+PathFind.prototype.notify_update = function(message) {
```
Comment (geertweening) [inline src/core/pathfind.js]: should this be camelCased?

File: src/core/pathfind.js [hunk 2]
```diff
@@ -74,7 +78,7 @@ PathFind.prototype.notify_update = function (message) {
   }
 };
 
-PathFind.prototype.notify_superceded = function () {
+PathFind.prototype.notify_superceded = function() {
```
Comment (geertweening) [inline src/core/pathfind.js]: should this be camelCased as well?

Comment (clark800) [inline src/core/pathfind.js]: I only changed calls that were broken by removing the attachUnderscored converter. Changing all underscore functions would probably be a bigger project.

**Comment by geertweening** [comment]
LGTM

**Pull request merged**